### PR TITLE
[storage- azblob, azfile] Swagger update to stg93

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
- [x] The purpose of this PR is explained in this or a referenced issue.
- [] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [x] Tests are included and/or updated for code changes.
- [x] Updates to module CHANGELOG.md are included.
- [x] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

### CODE REVIEW

File: sdk/storage/azblob/internal/generated/zz_appendblob_client.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func (client *AppendBlobClient) appendBlockCreateRequest(ctx context.Context, co
 	if modifiedAccessConditions != nil && modifiedAccessConditions.IfTags != nil {
 		req.Raw().Header["x-ms-if-tags"] = []string{*modifiedAccessConditions.IfTags}
 	}
-	req.Raw().Header["x-ms-version"] = []string{ServiceVersion}
+	req.Raw().Header["x-ms-version"] = []string{"2024-05-04"}
```
Comment (souravgupta-msft) [inline sdk/storage/azblob/internal/generated/zz_appendblob_client.go:118]: Update the `ServiceVersion` constant.

File: sdk/storage/azblob/internal/generated/autorest.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ go: true
 clear-output-folder: false
 version: "^3.0.0"
 license-header: MICROSOFT_MIT_NO_VERSION
-input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a32d0b2423d19835246bb2ef92941503bfd5e734/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json"
+input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/cc0d4436a4dd7b09c3dfd95e5ab7d6ab1e1eb17b/specification/storage/data-plane/Microsoft.BlobStorage/preview/2024-05-04/blob.json"
```
Comment (tanyasethi-msft) [inline sdk/storage/azblob/internal/generated/autorest.md:10]: To be updated from Main Branch

**Pull request merged**